src/pages/Lend/LoanChannelCategoryControl.vue

```diff
@@ -375,6 +377,7 @@ export default {
 		}
 	},
 	created() {
+		this.loading = true;
```

Comment:
When we land here from the experiment layout, the component won't prefetch, this loading will set the loading state to true. Then base data will miss the cache, so the loading state will stay on until the observer finished updating and loads the loans. loading is then clears in the `activateLoanChannelWatchQuery` next() function below. 

This addition adds the loader when coming from the experiment layout and the text "No loans" is displayed. It does not seem to affect normal operation of the page (coming from a link, with prefetch)